### PR TITLE
Add Toki Network and Toki Testnet

### PR DESCRIPTION
**Pull request by evanwang0**

### CODE REVIEW

**Review by ligi — CHANGES_REQUESTED** [review]
short name must be unique - but found `TOKI` more than once
also rebase to master please
and the icon should have `width` and `height` fields

**Comment by evanwang0** [comment]
Fixed. I modified the README slightly, because it is inconsistent. Please advise if the copy should be adjusted.

File: _data/icons/toki.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "url": "ipfs://QmZzDBbiideNFaNFxfHxeuB6Y5uTC4LMpaoHBtpzBj9Qco",
+    "width": "512",
+    "height": "512",
```
Comment (ligi) [inline _data/icons/toki.json:5]:
```suggestion
    "width": 512,
    "height": 512,
```

**Review by ligi — CHANGES_REQUESTED** [review]

File: _data/icons/toki.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "url": "ipfs://QmZzDBbiideNFaNFxfHxeuB6Y5uTC4LMpaoHBtpzBj9Qco",
+    "width": 512,
+    "height": 512,
```
Comment (ligi) [inline _data/icons/toki.json:5]: size does not seem to match:

```
identify QmZzDBbiideNFaNFxfHxeuB6Y5uTC4LMpaoHBtpzBj9Qco       07:54:22
QmZzDBbiideNFaNFxfHxeuB6Y5uTC4LMpaoHBtpzBj9Qco SVG 517x517 517x517+0+0 8-bit sRGB 12542B 0.000u 0:00.000

```

**Review by ligi — CHANGES_REQUESTED** [review]

File: _data/icons/toki.json (hunk 2 above)
Comment (evanwang0) [reply _data/icons/toki.json:5]: That's odd. The svg is below and does not have any reference to 517. It uses a viewBox but does not explicitly set a width/height - making it dimensionless. What tool are you using to print the above? I can make sure that it matches.

Comment (evanwang0) [reply _data/icons/toki.json:5]: I've updated the commit. If this does not work, please let me know the tool generating the above so I can troubleshoot.

**Comment by evanwang0** [comment]
Rebased

File: _data/chains/eip155-8654.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
+{
+  "name": "Toki Network",
+  "chain": "TOKI",
+  "network": "mainnet",
```
Comment (ligi) [inline _data/chains/eip155-8654.json:4]:
```suggestion
```

File: _data/chains/eip155-8655.json [hunk 4]
```diff
@@ -0,0 +1,20 @@
+{
+  "name": "Toki Testnet",
+  "chain": "TOKI",
+  "network": "testnet",
+  "rpc": [
```
Comment (ligi) [inline _data/chains/eip155-8655.json:5]:
```suggestion
  "network": "testnet",
```

**Comment by evanwang0** [comment]
@ligi Remove network?

**Review by ligi — CHANGES_REQUESTED** [review]
yea - and copy the other jsons schema / style introduced in #1509 - @ashutoshpw can you maybe add to the readme a section about this?

**Comment by ashutoshpw** [comment]
> yea - and copy the other jsons schema / style introduced in #1509 - @ashutoshpw can you maybe add to the readme a section about this?

Will be adding this soon.

**Comment by ashutoshpw** [comment]
This is auto-formatted if you have prettier extension installed in VS Code. If you don't have extension installed, you can format the files by running this command in project root directory. (You should have NodeJS installed in your system for this to run) 

`npx prettier --write --ignore-unknown _data`

**Comment by evanwang0** [comment]
@ashutoshpw I formatted the files after I saw the error in the CI. :)

**Review by ligi — APPROVED** [review]

**Pull request merged**